### ISSUE TITLE
Logger integration?

### ISSUE THREAD

**Issue by cs-victor-nascimento** [bryanjos/joken#69]
Most of the time I want to debug something in Joken I end up adding some logging. The Logger framework is awesome and I it is easy to integrate it in an application's config.

When disabling it in config, since all its functions are macros, they are not even included in the final AST. This is real neat!

It includes a per application configuration so we could assume a default to ERROR only and advise in the README that while developing it is good to enable it in a `:dev` config.

Thoughts?

**Comment by bryanjos** [comment]
I agree we can adding logging. It will not only help us, but also anyone debugging their applications. The only reason I never put in logging is because I think it can clutter up code, but that's not a very good reason if it's done right.

**Comment by cs-victor-nascimento** [comment]
I'd add Logging as debug on verifying and signing for now. That is as simple as:

```elixir
require Logger
Logger.debug fn -> "Verifying  #{inspect compact} using #{inspect signer}" end
Logger.debug fn -> "Signing #{inspect token} using #{inspect signer}" end
```

**Comment by cs-victor-nascimento** [comment]
Fixed in #81.

All we need is to discuss if we should log the key or not.
